tests/Common/TriggersTest.php

```diff
@@ -543,33 +564,77 @@ public function testDeleteTriggerWithNonMasterToken()
             'tableIds' => [$table1],
         ]);
 
+        $newNonAdminTokenWithoutPermission = $this->tokens->createToken((new TokenCreateOptions()));
+        $clientWithoutAdminTokenWithoutPermission = $this->getClient(['url' => STORAGE_API_URL, 'token' => $newNonAdminTokenWithoutPermission['token']]);
 
-        $anotherToken = $this->tokens->createToken($optionsForMainToken);
-        $anotherClientWithAnotherToken = $this->getClient([
-            'url' => STORAGE_API_URL,
-            'token' => $anotherToken['token'],
-        ]);
-
+        // try to delete the trigger with non-master token without required permission
         try {
-            $anotherClientWithAnotherToken->deleteTrigger((int) $trigger['id']);
+            $clientWithoutAdminTokenWithoutPermission->deleteTrigger((int) $trigger['id']);
             self::fail('should fail');
         } catch (Exception $e) {
-            self::assertEquals('Insufficient privilege. This token cannot manipulate with this trigger', $e->getMessage());
+            self::assertEquals('Your token does not have sufficient privilege.', $e->getMessage());
         }
 
-        // owner can delete it even isn't master token
-        $clientWithoutAdminToken->deleteTrigger((int) $trigger['id']);
+        $anotherToken = $this->tokens->createToken($optionForToken);
+        $anotherClientWithAnotherToken = $this->getClient([
+            'url' => STORAGE_API_URL,
+            'token' => $anotherToken['token'],
+        ]);
 
-        $this->expectException(ClientException::class);
-        $this->expectExceptionMessage(sprintf('Trigger with id [%d] was not found', $trigger['id']));
-        $clientWithoutAdminToken->getTrigger((int) $trigger['id']);
+        // trigger can be deleted by non-master token with right privilege
+        $anotherClientWithAnotherToken->deleteTrigger((int) $trigger['id']);
+        $triggerList = $clientWithoutAdminToken->listTriggers([$table1]);
+        $this->assertNotContainsTriggerId($trigger['id'], $triggerList);
+        $this->assertCount(1, $triggerList);
 
         // master token can delete it anyway
         $this->_client->deleteTrigger((int) $trigger2['id']);
+        $triggerList = $clientWithoutAdminToken->listTriggers([$table1]);
+        $this->assertNotContainsTriggerId($trigger2['id'], $triggerList);
 
-        $this->expectException(ClientException::class);
-        $this->expectExceptionMessage(sprintf('Trigger with id [%d] was not found', $trigger2['id']));
-        $this->_client->getTrigger((int) $trigger2['id']);
+        $this->assertCount(0, $triggerList);
+    }
+
+    /**
+     * @dataProvider tokenCreateOptionsProvider
+     * @return void
+     */
+    public function testDeleteTriggerCreatedByNonMasterToken(TokenCreateOptions $optionForToken)
```

Comment:
Tohle nechápu, co testuje. To testuje, že když to může vytvořit, tak to může i smazat? To mi skoro přijde nadbytečné. 

Možná to aspoň přejmenovat
```suggestion
    public function testNonMasterTokenCanDeleteTriggerItCreated(TokenCreateOptions $optionForToken)
```